pkg/selector/selector.go

```diff
@@ -391,3 +392,18 @@ func isInAllowList(allowRegex *regexp.Regexp, instanceTypeName string) bool {
 	}
 	return allowRegex.MatchString(instanceTypeName)
 }
+
+func gbToMBRange(gbRange *Float64RangeFilter) *IntRangeFilter {
```

Comment:
gibToMibRange? little confusing that this is the only place where you switch the syntax

Reply:
I was going back-and-forth on this, I thought it looked funny, but you're right it should be consistent.